src/components/Onboarding/steps/Analytics.js

```diff
@@ -67,13 +69,15 @@ class Analytics extends PureComponent<StepProps, State> {
     const { nextStep, t, onboarding } = this.props
     const { analyticsToggle, sentryLogsToggle } = this.state
 
+    const model = getDeviceModel(onboarding.deviceModelId)
```

Comment:
which creates a type problem `getDeviceModel` assume you give an id that is not `""` so either we need to remove this case, or have an invariant? at this step we should have it filled but it's hard to express it as a type I imagine?